packages/api/src/validations/validations.ts

```diff
@@ -571,7 +571,7 @@ export const validateWith = (func: () => void) => {
     func()
   } catch (e) {
     const message = (e as Error).message || (e as string)
-    throw new ValidationErrors.ServiceValidationError(message)
+    throw new ValidationErrors.ServiceValidationError(message, message)
```

Comment:
Hmm, isn't the second arg to `ServiceValidationError` an object for string substitutions? How does this work if you just pass the `message` string twice?

Reply:
Oh that may be a mistake -- I originally had code that passed two messages: a main and field one, but that got too complicated for this round to manage (and would change the signature of the validate, so I abandoned).